server/src/controllers/course.ts

```diff
@@ -26,7 +26,7 @@ export interface LocalizedString {
 
 export interface CourseData {
   // course id is either number type id in grades db or undefined when representing parsed sisu data
-  id?: number | undefined,
```

Comment:
Just to ensure I understand, this is still meant to be optional, but the `| undefined` is being removed due to redundancy?

Reply:
Resolved, thanks @no-comma